src/main/java/uk/gov/hmcts/cmc/claimstore/services/staff/DefendantResponseStaffNotificationService.java

```diff
@@ -61,19 +73,40 @@ public void notifyStaffDefenceSubmittedFor(
         );
     }
 
+    private boolean isFullAdmission(Claim claim) {
+        ResponseType responseType = claim.getResponse().orElseThrow(IllegalArgumentException::new).getResponseType();
+        return responseType == ResponseType.FULL_ADMISSION;
+    }
+
     public static Map<String, Object> wrapInMap(
         Claim claim,
         String defendantEmail
     ) {
         Map<String, Object> map = new HashMap<>();
+
+        Response response = claim.getResponse().orElseThrow(IllegalStateException::new);
         map.put("claim", claim);
-        map.put("response", claim.getResponse().orElseThrow(IllegalStateException::new));
+        map.put("response", response);
         map.put("defendantEmail", defendantEmail);
-        map.put("defendantMobilePhone", claim.getResponse()
-            .orElseThrow(IllegalStateException::new)
-            .getDefendant()
-            .getMobilePhone()
-            .orElse(null));
+
+        switch (response.getResponseType()) {
+            case FULL_DEFENCE:
+                map.put("defendantMobilePhone", response
```

Comment:
That should be regardless response type as this is just part of the task where defendant confirms their details independent from response type.

Reply:
This is not required for the email template for full admission, only needed in full defence.

Reply:
I understand it, although IMO content provider should map every single case details and make it available for every single PDF. I would like to have either one content provider for case or none.

I asked Art question why the following is missing in full admission email body:

```
Defendant's details can be found on the attached defence copy. Additional contact details are:

Email: {{ defendantEmail }}

{% if defendantMobilePhone is not empty %}
Mobile number: {{ defendantMobilePhone }}
{% else %}
Mobile number: not given
{% endif %}
```

Reply:
@sabahirfan despote this not being in the story Art confirmed that we should have it in the email body regardless of response type.

Reply:
added